data/strings/types_strings.txt

```diff
@@ -25449,7 +25210,7 @@
     zh-Hans = 旅游要素
 
   [type.tourism.alpine_hut]
-    en = Mountains Lodging
+    en = Mountain Lodging
```

Comment:
Alpine Hut would really be better.

Reply:
how about a compromise with 'Alpine Lodging' :)

Reply:
We call them huts hear and you have to sign up, but they are for the public.   Lodging has the connotation of more than a bunk in a shelter where you can put your sleeping bag.   However it's a good compromise and people are unlikely to be confused about a place high in the mountains that can only be reached on foot.